apps/src/sites/studio/pages/scripts/edit.js

```diff
@@ -7,23 +7,24 @@ import {getStore, registerReducers} from '@cdo/apps/redux';
 import isRtl from '@cdo/apps/code-studio/isRtlRedux';
 import getScriptData from '@cdo/apps/util/getScriptData';
 import reducers, {
-  init
+  init,
+  empty_non_user_facing_group
 } from '@cdo/apps/lib/levelbuilder/script-editor/scriptEditorRedux';
 import ScriptEditor from '@cdo/apps/lib/levelbuilder/script-editor/ScriptEditor';
 import {valueOr} from '@cdo/apps/utils';
 
 export default function initPage(scriptEditorData) {
   const scriptData = scriptEditorData.script;
   const lessonLevelData = scriptEditorData.lessonLevelData;
-  const lesson_groups = (scriptData.lesson_groups || [])
+  let lesson_groups = (scriptData.lesson_groups || [])
```

Comment:
please use camelCase for variable names, here and below. 

ideally, snake_case shouldn't make it any farther into our system than this "entry point". in cases where we have already let snake case into redux, no need to fix that right now, but in the future we may go back and try to push snake case further out of js.